RFC-0001-Agile Coretime.md

```diff
@@ -0,0 +1,291 @@
+# RFC-1: Agile Coretime
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Status**      | Draft Proposal                                                                              |
+| **Areas**       | General                                                                                     |
+| **Description** | Agile periodic-sale-based model for assigning Coretime on the Polkadot Ubiquitous Computer. |
+| **Issues**      | n/a                                                                                         | 
+| **Authors**     | Gavin Wood                                                                                  |
+| **Reviewers**   | None                                                                                        |
+
+
+## Summary
+
+This proposes a periodic, sale-based method for assigning and Polkadot Coretime. The method takes into account the need for long-term capital expenditure planning for teams building on Polkadot, yet also provides a means to allow Polkadot to capture long-term value in the resource which it sells. It supports the possibility of building secondary markets to make resource allocation more efficient and largely avoids the need for parameterisation.
+
+## Motivation
+
+### Present System
+
+The present system of allocating time for parachains on the cores of the Polkadot Ubiquitous Computer (aka "Polkadot") is through a process known as *slot auctions*. These are on-chain candle auctions which proceed for several days and result in a core being assigned to a single parachain for six months at a time up to 18 months in advance. Practically speaking, we only see two year periods being bid upon and leased.
+
+Funds behind the bids made in the slot auctions are merely locked, not consumed or paid and become unlocked and returned to the bidder on expirt of the lease period. A means of sharing the deposit trustlessly known as a *crowdloan* is available allowing token holders to contribute to the overall deposit of a chain without any counterparty risk.
```

Comment:
> Funds behind the bids made in the slot auctions are merely locked, not consumed or paid

This wording and multiple other references to sales/ purchase suggest that Bulk coretime will be *paid for* in DOT, in contrast to currently where it's simply locked and the "cost" is the opportunity cost of not staking - it should be explicit if this is the case.

It would also be useful to understand - if this is the case - where those DOT are sent, i.e. who is paid? Is it validators? Is it burned? If validators, would high reward rates through demand for blockspace impact inflation that currently forms the vast majority of their revenue?

Reply:
> Bulk coretime will be paid for in DOT

That's right.

Any DOT recuperated for sales of system resources (Coretime, in this case) would by default be placed in the treasury. Governance would be able to determine what to do with it.

Reply:
Maybe its better to burn the DOT instead of diverting them to Treasury. There are a few reasons for that:
1) In the absence of 2y locks on DOT, the system might benefit from a permanent sink for DOTs. We also might consider to increase the ideal staking rate. Non-interactive staking might serve well here, too.
2) Inflow from coretime usage might, especially in the short-term, be very volatile. The Treasury conceptually benefits from predictable inflow, allowing for long-term budgeting. Inflation is the best way to do that. We'd counter that with burning for coretime.
3) This mechanism would lead to high inflow in times with high coretime usage and low inflow in times of low usage. It seems to me that, if anything, it should be the opposite. With a steady inflow the Treasury would always have enough funds to respond to demand shocks in coretime when necessary (by funding good projects / initiatives).

Reply:
I totally agree with @jonasW3F , DOT from Coretime sales should be instantly burnt in order to slow down inflation.